packages/@react-spectrum/actionbar/src/actionbar.css

```diff
@@ -13,57 +14,70 @@
 }
 
 .react-spectrum-ActionBar {
-  --spectrum-padding: var(--spectrum-global-dimension-size-100);
-  --column-gap: var(--spectrum-actionbuttongroup-text-button-gap-x);
-  --row-gap: var(--spectrum-actionbuttongroup-text-button-gap-y);
-  --show-transition-duration: var(--spectrum-global-animation-duration-200);
-  --show-transition-timing: var(--spectrum-global-animation-linear);
+  transition: height var(--spectrum-global-animation-duration-100) ease-in-out,
+              opacity var(--spectrum-global-animation-duration-100) ease-in-out;
 
-  height: 0;
+  /* Take up no space when not open */
+  block-size: 0;
+  opacity: 0;
+  width: 100%;
+
+  /* Clip anything outside of us, whether we're open or not. This makes the animation smooth */
   overflow: hidden;
-  transition-property: height;
-  transition-duration: var(--show-transition-duration);
-  transition-timing-function: var(--show-transition-timing);
 
-  background: white;
+  &.is-open {
+    height: calc(var(--spectrum-global-dimension-size-600) + var(--spectrum-global-dimension-size-100) * 2);
+    opacity: 1;
+    transition-delay: 0ms;
+  }
 
-  & > div {
+  .react-spectrum-ActionBar-bar {
+    display: grid;
+    grid-template-areas: "clear selected . actiongroup";
+    grid-template-columns: auto max-content minmax(var(--spectrum-global-dimension-size-300), 1fr) auto;
```

Comment:
I know we talked about this a little bit, but just for my memory, did `auto 1fr var(--spectrum-global-dimension-size-300) auto` not work?

Reply:
Can this be a followup to try out?